packages/smart/test/turbo-test.ts

```diff
@@ -88,6 +88,13 @@ describe("Turbo", () => {
     [turboId] = log.args;
     expect(turboId).to.equal(0);
 
+    const data = await arbiter.getEncodedTurboData(turboId.toNumber());
+    const turboData = await arbiter.decodeTurboData(data);
+    console.log("turboData", turboData);
+
+    // Checking one random property should be sufficient.
+    expect(turboData[3]).to.equal(numTicks);
```

Comment:
ought to check one of the array properties since those are the ones being dropped